tardis/io/logger/logger.py

```diff
@@ -85,7 +86,7 @@ def logging_state(log_level, tardis_config, specific):
         Allows to set specific logging levels. Logs of the `log_level` level would be output.
     """
 
-    if "debug" in tardis_config:
+    if "debug" in tardis_config and "specific" in tardis_config["debug"]:
         specific = (
             tardis_config["debug"]["specific"] if specific is None else specific
```

Comment:
If `specific` is undefined, then perhaps it should be `else DEFAULT_SPECIFIC_STATE`